third_party/wabt/src/interp/jit/jit-backend.cc

```diff
@@ -120,6 +128,40 @@ static sljit_uw SLJIT_FUNC getTrapHandler(sljit_uw return_addr) {
   return trap_handler_list.get(return_addr);
 }
 
+// Float operations.
+// TODO Canonical NaN
+static sljit_f32 floatNearest(sljit_f32 val) {
+  return std::nearbyint(val);
+}
+
+static sljit_f64 floatNearest(sljit_f64 val) {
+  return std::nearbyint(val);
+}
+
+static sljit_f32 floatMin(sljit_f32 lhs, sljit_f32 rhs) {
+  return std::min(lhs, rhs);
+}
+
+static sljit_f64 floatMin(sljit_f64 lhs, sljit_f64 rhs) {
+  return std::min(lhs, rhs);
+}
+
+static sljit_f32 floatMax(sljit_f32 lhs, sljit_f32 rhs) {
+  return std::max(lhs, rhs);
```

Comment:
Wabt is doing this in a more complex way:
https://github.com/Samsung/walrus/blob/main/third_party/wabt/include/wabt/interp/interp-math.h#L267